File: app/src/main/java/demo/com/stickerapp/MainActivity.java
package demo.com.stickerapp;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import java.util.Date;

import javax.inject.Inject;

import demo.com.stickerapp.data.model.MyExample;

public class MainActivity extends AppCompatActivity {

    @Inject
    MyExample mMyExample;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView dateTextView = findViewById(R.id.textView);
        ((MyApplication) getApplication())
                .getMyComponent()
                .inject(MainActivity.this);
        dateTextView.setText((new Date(mMyExample.getDate())).toString());
    }

}

File: app/src/main/java/demo/com/stickerapp/MainActivity.kt
package demo.com.stickerapp

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import demo.com.stickerapp.data.model.User

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var user = User()
        user.id = 10;
        println(user.id)
    }
}
